packages/axios-extension/src/abap/catalog/v2-catalog-service.ts

```diff
@@ -219,4 +221,25 @@ export class V2CatalogService extends CatalogService {
         }
         return annotations;
     }
+
+    /**
+     * Calls endpoint `ServiceTypeForHUBServices` to determine the service type.
+     *
+     * @param path service path
+     * @returns service type
+     */
+    public async getServiceType(path: string): Promise<ServiceType> {
```

Comment:
Should this be returning `Promise<ServiceType | undefined>` as `serviceType` is undefined by default in the method?

Reply:
Thanks @longieirl I've updated